serif_shared/src/commonMain/kotlin/xyz/room409/serif/serif_shared/Database.kt

```diff
@@ -38,4 +39,25 @@ object Database {
     fun deleteAllSessions() {
         this.db?.sessionDbQueries?.deleteAllSessions()
     }
+
+    fun getImageInCache(url: String): String? {
+        val cached_img = this.db?.sessionDbQueries?.selectCachedMedia(url) { mxcUrl: String, localPath: String ->
```

Comment:
Unnecessary variable?

Reply:
Correct, we could just return the expression, I should change that in the audio branch.